docs/_articles/en/user-guide/byotemplate.md

```diff
@@ -0,0 +1,73 @@
+|apextrigger    | ApexTrigger.trigger <br> _trigger.trigger-meta.xml
+|Lightningapp   |DefaultLightningApp.app <br> DefaultLightningController.js <br> DefaultLightningCss.css <br> DefaultLightningHelper.js <br> DefaultLightningRenderer.js <br> DefaultLightningSVG.svg <br> DefaultLightningAuradoc.auradoc <br>auradefinitionbundle.app-meta.xml
+|Lightningcomponent |aura/default/default.cmp <br> aura/default/default.cmp-meta.xml <br> aura/default/defaultController.js <br> aura/default/defaultHelper.js <br> aura/default/defaultRenderer.js <br> aura/default/default.svg <br> aura/default/default.design <br> aura/default/default.css <br> aura/default/default.auradoc <br> lwc/default/default.js<br> lwc/default/default.html <br> lwc/default/default.js-meta.xml |
+|Lightningevent | defaultLightningEvt.evt <br> auradefinitionbundle.evt-meta.xml
+|Lightninginterface | DefaultLightningIntf.intf <br> auradefinitionbundle.intf-meta.xml
+|Lightningtest | DefaultLightningTest.resource
+|Project | Any file in the project folder
+|Staticresource | empty.resource <br> empty.js <br> empty.json <br> empty.css <br> empty.txt <br> _staticresource.resource-meta.xml
+|visualforcecomponent| DefaultVFComponent.component <br> _component.component-meta.xml
+|visualforcepage | DefaultVFPage.page <br> _page.page-meta.xml
+
+## Clone the Repo
+See GitHub documentation for instructions on how to clone repos.
+
+## Set Default Template Location
+1. Open the `sfdx-config.json` config file in your `<project-folder>/.sfdx` folder
+2. Add the parameter `customOrgMetadataTemplates`, and set its value to either a local directory or your cloned GitHub repository that contains your default templates. The GitHub URL can either point to the root folder that contains your templates, or a path to a subfolder on a branch in the repo that contains your templates. For example:
+```
+{
+"customOrgMetadataTemplates": ""https://github.com/mygithubacct/salesforcedx-templates""
+}
+```
+**Note:** You can also use CLI to set this parameter. See [CLI Runtime Configuration Values](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_dev_cli_config_values.htm) for more information.
+
+## Create a Apex Class with Custom Code
+1. Clone the sample GitHub repo.
+2. Delete all folders except the `apexclass` folder. Also delete all the files except the `DefaultApexClass.cls` file from the `apexclass` folder.
+3. Edit the `DefaultApexClass.cls` file with your custom code:
+
+```
+public with sharing class <%= apiName %> {
+    		public <%= apiName %>(String prop) {
+			this.prop = prop;
+    		}
+
+		@AuraEnabled public String prop { get;set; }
+	}
+``` 
+**Note:** If you edit the file locally, remember to push changes to your repo. You can also choose to make changes directly in your repo. 
+
+4. Update the `sfdx-config.json` file to point to your cloned repo (or local directory).
+5. Run the `SFDX: Create Apex Class` command from the Command Palette.
```

Comment:
Do we use bold in this case for a menu command? I'd have to go look at some of our other topics.

Reply:
I checked a few places (https://github.com/forcedotcom/salesforcedx-vscode/pull/3250/files) and it looks like we don't use bold...